functests/6_mustgather_testing/test_suite_mustgather_test.go

```diff
@@ -0,0 +1,61 @@
+package pao_mustgather
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/junit"
+
+	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
+)
+
+var _ = BeforeSuite(func() {
+	By("Looking for oc tool")
+	ocExec, err := exec.LookPath("oc")
+	if err != nil {
+		fmt.Fprintf(ginkgo.GinkgoWriter, "Unable to find oc executable: %v\n", err)
+		Skip(fmt.Sprintf("unable to find 'oc' executable %v\n", err))
+	}
+
+	mgImage := "quay.io/openshift-kni/performance-addon-operator-must-gather"
+	mgTag := "4.11-snapshot"
```

Comment:
notice that you are using here the 4.10 branch

Reply:
right. Fixed.